api/spec/requests/api/v1/twitter_queries_spec.rb

```diff
@@ -23,7 +23,7 @@
                         model: TwitterQuery,
                         authorized_user: :admin,
                         tags: "Projects",
-                        summary: "Gets all twitter queries associated with a project",
+                        summary: "Returns all twitter queries associated with a project",
```

Comment:
This change was made to be more consistent with the other summaries in the documentation.